README.md

```diff
@@ -1,8 +1,31 @@
 # CSS Learning Resources
+
 A collection of the best CSS resources. 
 
+## Websites for CSS
+
+- https://www.w3schools.com/css/
```

Comment:
Thank you for all of the additions. Could you please provide an updated link similar to this `* [CSS for JavaScript Developers](https://css-for-js.dev/)` so that it will have a more user-friendly name in the square brackets to link to the url in parens instead?

Reply:
OKkh i will do that